sig-and-wg/wg-knative/knative-eventing-adoption.md

```diff
@@ -122,28 +122,46 @@ To enable ease of use and better UX, it should be possible to have:
 
 * A default PubSub.   `Lowest Priority`
 * A PubSub per application
-* A PubSub specified for an event type during registration. `Highest Priority`
+* A PubSub specified for an Event type during registration. `Highest Priority`
 
 **Requirements:**
 
 * API to expose available PubSub in Kyma.
 
+**Flow**
+![](assets/decide-which-pubsub.svg)
+
```

Comment:
Let's add this introductory line above the diagram:

`The diagram presents the process of deciding which PubSub to use.`